$NetBSD: patch-build_cmake_utils_CMakeLists.txt,v 1.1 2022/07/28 05:25:22 dbj Exp $

Fix use of DESTDIR
https://github.com/wxWidgets/wxWidgets/issues/22610

Respect ${CMAKE_INSTALL_BINDIR} instead of assuming bin

--- build/cmake/utils/CMakeLists.txt.orig	2022-07-28 04:31:27.000000000 +0000
+++ build/cmake/utils/CMakeLists.txt
@@ -26,21 +26,22 @@ if(wxUSE_XRC)
     set_target_properties(wxrc PROPERTIES FOLDER "Utilities")
 
     wx_install(TARGETS wxrc
-        RUNTIME DESTINATION "bin"
-        BUNDLE DESTINATION "bin"
+        RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}"
+        BUNDLE DESTINATION "${CMAKE_INSTALL_BINDIR}"
         )
 
-    if(NOT WIN32_MSVC_NAMING)
+    if(NOT WIN32_MSVC_NAMING AND wxBUILD_INSTALL)
         if(IPHONE)
             set(EXE_SUFFIX ".app")
         else()
             set(EXE_SUFFIX ${CMAKE_EXECUTABLE_SUFFIX})
         endif()
 
-        wx_install(CODE "execute_process( \
+        # Don't use wx_install() here to preserve escaping.
+        install(CODE "execute_process( \
             COMMAND ${CMAKE_COMMAND} -E create_symlink \
-            ${CMAKE_INSTALL_PREFIX}/bin/${wxrc_output_name}${EXE_SUFFIX} \
-            ${CMAKE_INSTALL_PREFIX}/bin/wxrc${EXE_SUFFIX} \
+            ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_BINDIR}/${wxrc_output_name}${EXE_SUFFIX} \
+            \"\$ENV{DESTDIR}${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_BINDIR}/wxrc${EXE_SUFFIX}\" \
             )"
         )
     endif()
